Safewaan/HonkedIn
# Organize Project Structure- Create New Directories for Pages and APIs

## Implementation description
* Created new directory, `/src/routes/` to store all APIs within the `/src/` directory
* Created new directories in `/src/components/` to store pages neatly
    `auth`: user authentication related, generally related to pages that use Firebase heavily
    `common`: components we create that are reusable
    `constants`: files that store constants
    `events`: event related pages
    `forums`: forum related pages
    `users`: user related pages
* Created new directories in `/routes/` to categorize APIs that call data by specified table
* Created new file, `src/components/constants/Routes.js` to store page routes. Updated all page references to use these routes
* Created separate directory, `./tests/` for test related files
* Added readme for steps on how to create new routes


## Steps to test
1. Verify all endpoints and APIs are functioning as intended
2. Verify all pages can be accessed, and redirect to the correct page
3. Verify that cypress and e2e tests are passing


## What should reviewers focus on?
* Is the new structure intuitive? Are there any changes that should be made?
